beets/ui/commands.py

```diff
@@ -252,7 +251,7 @@ def format_index(track_info):
         print_("To:")
         show_album(artist_r, album_r)
     else:
-        print_(u"Tagging:\n    {0.artist} - {0.album}".format(match.info))
+        print_(u"Tagging:\n    {0}".format(match.info))
```

Comment:
Here, `match.info` is not an Album or an Item; it's an AlbumInfo or an ItemInfo (i.e., a potential match). We can't currently use formatting for this.